department-of-veterans-affairs/va.gov-cms
# VACMS-7649 add patch for text counter behavior in claro

## Description

Relates to #7649. Claro structures the textfields a little differently caused some conflicts with the textfield_counter module. There is a patch in a merge request here: https://www.drupal.org/project/textfield_counter/issues/3182675#comment-14380796 that works perfectly for us. Have also commented it as RTBC, hopefully it makes it in & we can take the patch out with a module upgrade in the future.

## Testing done
- Verified text counter in vagovclaro works as expected for all fields that have a text counter. Tested on text fields, link summary, textareas, and rich text paragraph with the character limit defined.
- Verified text counter works as expected in the vagovadmin theme

node/33340/edit

## Screenshots
![Screen Shot 2022-01-20 at 2 05 00 PM](https://user-images.githubusercontent.com/11279744/150429700-b660a683-c5cc-4313-8714-0763f74c1243.png)
![Screen Shot 2022-01-20 at 2 05 13 PM](https://user-images.githubusercontent.com/11279744/150429704-9c1fc860-e956-40b3-95e1-93aef4c5d619.png)


## QA steps
- log in, check `node/33340/edit` and verify that text counter works as expected

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
